ACLPManager/manager
# DI-20800 - Beta review and UI feedbacks

## Description 📝
This PR contains the CSS and UI functionality fixed according to UX beta feedbacks

## Changes  🔄
List any change relevant to the reviewer.

1. Added Tooltips for widget level filters and icons
2. Added publishing the resource selection only on close of the popup
3. Adjusted the padding and removed the height property of legend row in widget
4. CamelCase for aggregation filters

## Target release date 🗓️
10-10-2024

## Preview 📷
**Include a screenshot or screen recording of the change**

:bulb: Use `<video src="" />` tag when including recordings in table.

| Before  | After   |
| ------- | ------- |
| 📷 | 📷 |

## How to test 🧪

1. Login as mock user
2. Go to monitor page
3. Select a dashboard and all mandatory filters
4. You should be able to see all the fixes mentioned above

## As an Author I have considered 🤔

*Check all that apply*
